src/shipping/components/OrderValue/OrderValue.tsx

```diff
@@ -111,47 +116,68 @@ export const OrderValue: React.FC<OrderValueProps> = ({
               </TableCell>
             </TableHead>
             <TableBody>
-              {channels?.map(channel => (
-                <TableRow key={channel.id}>
-                  <TableCell>
-                    <Typography>{channel.name}</Typography>
-                  </TableCell>
-                  <TableCell>
-                    <PriceField
-                      disabled={disabled}
-                      label={intl.formatMessage({
-                        defaultMessage: "Min Value"
-                      })}
-                      name={`minValue:${channel.name}`}
-                      value={channel.minValue}
-                      onChange={e =>
-                        onChannelsChange(channel.id, {
-                          ...channel,
-                          minValue: e.target.value
-                        })
-                      }
-                      currencySymbol={defaultCurrency}
-                    />
-                  </TableCell>
-                  <TableCell>
-                    <PriceField
-                      disabled={disabled}
-                      label={intl.formatMessage({
-                        defaultMessage: "Max Value"
-                      })}
-                      name={`maxValue:${channel.name}`}
-                      value={channel.maxValue}
-                      onChange={e =>
-                        onChannelsChange(channel.id, {
-                          ...channel,
-                          maxValue: e.target.value
-                        })
-                      }
-                      currencySymbol={defaultCurrency}
-                    />
-                  </TableCell>
-                </TableRow>
-              ))}
+              {channels?.map(channel => {
+                const minError = formErrors.minimumOrderPrice?.find(error =>
```

Comment:
Since this piece of logic happens 3 times across the app, maybe we could extract to a helper function like so:
```
const getChannelError = (type, channelId) => 
    formErrors.[type]?.find(error =>
        error.channels?.find(id => id === channelId)
    );
```

Reply:
☝️ just type arguments